### ISSUE TITLE
Possible to connect to alternate *domain*?

### ISSUE THREAD

**Issue by hoodja** [zakird/pyad#38]
Hi,

I am trying to use pyad to automate some directory activities, but it seems no matter what I provide to pyad.set_defaults() I am always connected to the *production* domain. I operate alternative domains for testing environments and I am not sure what I can do to connect to those domains when testing my code.

I have tried passing the DNS name and the IP address of the development domain controller to set_defaults() as well as part of the options argument of specific calls but from what I can tell things are always resolved to the domain to which my user is currently attached.

**Comment by zakird** [comment]
What parameters are you supplying in set defaults?

**Comment by hoodja** [comment]
Hi Zakir,

I think I figured out my problem. My AD setup is "unique" in that we house
different environments under different top-level DCs. So while I was
attempting to connect to the "development environment" by using
set_defaults(), I should have instead been changing the DN I was using to
lookup objects.

Once I realized my disconnect and tried a few things the world made a lot
more sense to me than last week.

I'll close this issue - I don't think there is a need for anything here.
Sorry to bug you.

Thanks,
Jim
